open_humans/management/commands/migrate_request_sources.py

```diff
@@ -0,0 +1,74 @@
+from django.core.management.base import BaseCommand
```

Comment:
Adding a whole management command looks like a lot -- I'm sorry about the mess here, but is there any reason for me to not go in and fix the database right now?